### ISSUE TITLE
Error installing from github

### ISSUE THREAD

**Issue by bcow** [AndreaCirilloAC/updateR#1]
This project looks great! I wanted to test it updating from version 3.2.2 to 3.2.3.

Unfortunately, I'm getting an error right off the bat:
```
> install_github("andreacirilloac/updateR")
Downloading GitHub repo andreacirilloac/updateR@master
from URL https://api.github.com/repos/andreacirilloac/updateR/zipball/master
Installing updateR
'/Library/Frameworks/R.framework/Resources/bin/R' --no-site-file --no-environ --no-save --no-restore CMD INSTALL  \
  '/private/var/folders/st/17pmrjw918357qh9084cmt1m0000gn/T/Rtmpfua4so/devtoolsb211744b188d/AndreaCirilloAC-updateR-8f9f8fb'  \
  --library='/Library/Frameworks/R.framework/Versions/3.2/Resources/library' --install-tests 

* installing *source* package ‘updateR’ ...
** R
Error in parse(outFile) : 
  /private/var/folders/st/17pmrjw918357qh9084cmt1m0000gn/T/Rtmpfua4so/devtoolsb211744b188d/AndreaCirilloAC-updateR-8f9f8fb/R/up:1:97: unexpected ')'
1: updateR <- function(admin_password = "", page_source = "https://cran.rstudio.com/bin/macosx/" ) )
                                                                                                    ^
ERROR: unable to collate and parse R files for package ‘updateR’
* removing ‘/Library/Frameworks/R.framework/Versions/3.2/Resources/library/updateR’
Error: Command failed (1)
```

So I tried cloning the repo and building the package through Rstudio. Same error. 

I don't know enough about building packages to understand this error - but let me know if I can help with more testing!

**Comment by AndreaCirilloAC** [comment]
Thank you Betsy!

all the problem was coming from a redundant parenthesis: shame on me!

Thank you for pointing this out, let me know if everything is fine so that I can close this issue.

**Comment by AndreaCirilloAC** [comment]
closing this since it was a semantyc error which was fixed got fixed with #df2b5933b866cd078ba30e31a8b3fa128f668df8
thank you @bcow
